### PR TITLE
rabbitmq/plugins: fix for ansible 2.10.5

### PR DESCRIPTION
**Pull request by sk4zuzu**
- use "broker_state: offline"
- fix idempotence and performance

### CODE REVIEW

**Review by przemyslavic — DISMISSED** [review]

**Review by to-bar — DISMISSED** [review]
:umbrella:

**Comment by sk4zuzu** [comment]
Hahaha sorry guys, thanks for your patience! :hand_over_mouth:

File: core/src/epicli/data/common/ansible/playbooks/roles/rabbitmq/tasks/configure-rabbitmq.yml [hunk 1]
```diff
@@ -23,17 +23,24 @@
     dest: /usr/lib/rabbitmq/sbin
     state: link
 
+# RabbitMQ documentation (https://www.rabbitmq.com/rabbitmq-plugins.8.html) states:
+# > If rabbitmq-plugins is used on the same host as the target node,
+# > --offline can be specified to make rabbitmq-plugins resolve
+# > and update plugin state directly (without contacting the node).
+# > Such changes will only have an effect on next node start.
 - name: Enable plugins that were installed
+  when: specification.rabbitmq_plugins
```
Comment (to-bar) [inline core/src/epicli/data/common/ansible/playbooks/roles/rabbitmq/tasks/configure-rabbitmq.yml:32]:
```suggestion
  when:
    - specification.rabbitmq_plugins is defined
    - specification.rabbitmq_plugins
```

**Review by to-bar — APPROVED** [review]

**Review by przemyslavic — APPROVED** [review]

**Comment by przemyslavic** [comment]
@sk4zuzu merguj Pan to 💪

**Pull request merged**